assets/wizards/performance/views/intro/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Performance Wizard Intro screen
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Card, withWizardScreen } from '../../../../components/src';
+
+/**
+ * Intro screen for Performnance Wizard
+ */
+class Intro extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const { pluginRequirements } = this.props;
+		return (
+			<Fragment>
+				<Card>
+					<p>
+						{ __(
+							'Newspack utilizes PWA to automatically optomizing and configuring your news site to perform better. It improves:'
+						) }
+					</p>
+					<h2>{ __( 'Speed' ) }</h2>
+					<p>{ __( 'Will work reliably, no matter the network conditions.' ) }</p>
+					<h2>{ __( 'Security' ) }</h2>
+					<p>
+						{ __(
+							'Served from a secure origin through HTTPS, which will protect the integrity of your news site.'
+						) }
+					</p>
+					<h2>{ __( 'User Experience' ) }</h2>
+					<p>
+						{ __( 'Feels like a natural app on the device, with an immersive user experience.' ) }
+					</p>
+				</Card>
+				<Card>
+					<h2>{ __( 'Advanced Options' ) }</h2>
+					<p>
+						{ __(
+							'Increase engagement with your audience by implementing the following advanced options:'
+						) }
+					</p>
+					<ul>
+						<li>{ __( 'Add to homescreen' ) }</li>
+						<li>{ __( 'Offline usage' ) }</li>
+						<li>{ __( 'Push notifications' ) }</li>
+					</ul>
+				</Card>
+			</Fragment>
```

Comment:
Do we want a line like "Basic PWA options have been automatically set up for you. Advanced options are available."? I'm not sure whether anything happens/happened if I don't do the advanced options.

Reply:
This is a great idea, and I'd like to provide this type of message in any wizard that configures a plugin. Implemented here in https://github.com/Automattic/newspack-plugin/pull/141/commits/5e72e06fea2b8723928611e9bf90342692245d7c